hooks/useUser.tsx

```diff
@@ -19,12 +19,19 @@ const useUser = () => {
 
   useEffect(() => {
```

Comment:
If this is a component-level useEffect, be aware of any cleanup actions if they apply. Since these are one-time API requests, you’re likely fine, but it’s something to consider if you use it elsewhere.